tensorboard/webapp/plugins/npmi/util/metric_type_test.ts

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {
+  metricIsMetricCount,
+  metricIsNpmi,
+  metricIsNpmiAndNotDiff,
+  stripMetricString,
+} from './metric_type';
+
+describe('metric type utils', () => {
+  describe('metric is count', () => {
+    it('returns true if the metric starts with count@', () => {
```

Comment:
for thoroughness, add a test with input `foo@count` and/or `foo@count@` (to make sure it is not a substring match).